sys/net/gnrc/pktbuf/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_PKTBUF_STATIC
+    bool "Configure GNRC_PKTBUF"
+    depends on MODULE_GNRC_PKTBUF
```

Comment:
If this will only expose options for the static pktbuf, then I think it should depend on it instead:
`depends on MODULE_GNRC_PKTBUF_STATIC`